.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
![image](https://github.com/momo-87/math-magicians/assets/93039359/de8f1b94-f5cc-40ac-8cff-f6f7a5a75f64)


**[OPTIONAL]**

- [ ] It would be nice if you delete the highlighted title because you already have it after the Table of Contents 👍 to make your readme descriptive and professional 😊